sklearn/utils/metaestimators.py

```diff
@@ -23,7 +23,7 @@ def __init__(self):
         pass
 
     def _get_params(self, attr, deep=True):
-        out = super(_BaseComposition, self).get_params(deep=False)
+        out = super(_BaseComposition, self).get_params(deep=deep)
```

Comment:
This was needed to get the tests passing? (just to be sure this change is covered by the tests)

Reply:
This was updated to support `ColumnTransformer.get_params('remainder__{}')` where the  remainder is a estimator. This is tested with `test_column_transformer_get_set_params_with_remainder`.

Reply:
OK!